include/cute/atom/copy_traits_xe.hpp

```diff
@@ -191,24 +191,24 @@ CUTE_HOST_DEVICE auto prefetch_selector(TiledCopy<TiledCopyArgs...> const& tiled
 }
 
 
-template <class CopyOp, class StrideIndicator = cute::Stride<int64_t, cute::Int<1>, int64_t>>
+template <class CopyOp, class StrideOrTensor = cute::Stride<int64_t, cute::Int<1>, int64_t>>
 struct XE_2D_LD_Unpack {
-
   using BlockShape = typename CopyOp::BlockShape;
-  using Traits_LD_t = Copy_Traits<CopyOp, StrideIndicator>;
+  using Traits_LD_t = Copy_Traits<CopyOp, StrideOrTensor>;
 
-  static constexpr auto stride_rank = rank(StrideIndicator{});
+  static constexpr auto stride_rank = rank(StrideOrTensor{});
   static_assert(stride_rank == 2 || stride_rank == 3);
 
-  // Assume LD_T/LD_N will indicate ColumnMajor and RowMajor
-  static constexpr bool is_column_major = detail::is_transpose_load<CopyOp>;
+  // Assume LD_T/LD_N will be use column/row major matrices respectively
+  static constexpr bool is_transpose_copy = detail::is_transpose_load<CopyOp>;
 
-  // We need reverse some parameters becasue intel xe 2d copy intrinsic always assume the matrix is (M, N):(N, 1) convention
-  static constexpr bool is_need_reversed = detail::is_stride_leftmost<StrideIndicator>;
+  // We need reverse some parameters becasue intel xe 2d copy intrinsic always assume the matrix use (M, N):(N, 1) - row major convention
```

Comment:
I think `(M, N):(N, 1) - row major convention` is confusing because it implies that `(M, N):(N, 1)` represents a row major convention. This is only true for A & C.